templates/general/positions.html

```diff
@@ -0,0 +1,86 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <!-- Main component for a primary marketing message or call to action -->
+    <div class="jumbotron">
+        <h1>Positions</h1>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-heading">Executive Council Positions</div>
+        <div class="panel-body">
+            <table class="table table-condensed">
+                <thead style="font-weight:bold;">
+                <tr>
+                    <td>Position:</td>
+                    <td>Brother(s):</td>
+                    <td>Phone Number:</td>
+                </tr>

```

Comment:
Should the <tr> tag be indented? I noticed it is inside the <tbody> tag but not in the <thead> tag. I would keep indentation consistent between the two unless there is a reasoning behind it.